boards/px4/fmu-v6c/default.px4board

```diff
@@ -10,7 +10,7 @@ CONFIG_DRIVERS_BAROMETER_MS5611=y
 CONFIG_DRIVERS_BATT_SMBUS=y
 CONFIG_DRIVERS_CAMERA_CAPTURE=y
 CONFIG_DRIVERS_CAMERA_TRIGGER=y
-CONFIG_DRIVERS_DIFFERENTIAL_PRESSURE_ASP5033DRIVER=y
+CONFIG_DRIVERS_DIFFERENTIAL_PRESSURE_ASP5033=y
```

Comment:
We can't have this default for all 6C if that's not a common sensor used.

Reply:
Yes. I removed it from the default configuration for 6C.